readme.md

```diff
@@ -88,4 +88,10 @@ If you like this project, Give a ⭐️.
 - Project from [Microverse](https://www.microverse.org/?grsf=i6yi2m) RUBY, DATABASES module
 
 
```

Comment:
- [ ] To make your README to be more professional and descriptive, I think it would be better if you update the content of your `Getting Started` section with correct information. Right now,  some of info are wrong such as the name of your project in the cloning link,...
You can update it to this:
```
  cd my-preferred-folder

  git clone https://github.com/Talha-Hanif5666/school-library.git

  cd school-library
```